process/flow_hash.hpp

```diff
@@ -0,0 +1,133 @@
+/**
+ * \file flow_hash.hpp
+ * \brief Plugin for parsing flow_hash traffic.
+ * \author Jakub Antonín Štigler xstigl00@stud.fit.vut.cz
+ * \date 2023
+ */
+/*
+ * Copyright (C) 2023 CESNET
+ *
+ * LICENSE TERMS
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ * 1. Redistributions of source code must retain the above copyright
+ *    notice, this list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright
+ *    notice, this list of conditions and the following disclaimer in
+ *    the documentation and/or other materials provided with the
+ *    distribution.
+ * 3. Neither the name of the Company nor the names of its contributors
+ *    may be used to endorse or promote products derived from this
+ *    software without specific prior written permission.
+ *
+ *
+ *
+ */
+
+#ifndef IPXP_PROCESS_FLOW_HASH_HPP
+#define IPXP_PROCESS_FLOW_HASH_HPP
+
+#include <cstring>
+
+#ifdef WITH_NEMEA
+  #include "fields.h"
+#endif
+
+#include <string>
+#include <sstream>
+
+#include <ipfixprobe/process.hpp>
+#include <ipfixprobe/flowifc.hpp>
+#include <ipfixprobe/packet.hpp>
+#include <ipfixprobe/ipfix-elements.hpp>
+#include <ipfixprobe/byte-utils.hpp>
+
+namespace ipxp {
+
+#define FLOW_HASH_UNIREC_TEMPLATE "FLOW_ID"
+
+UR_FIELDS (
+    uint64 FLOW_ID
+)
+
+/**
+ * \brief Flow record extension header for storing parsed FLOW_HASH data.
+ */
+struct RecordExtFLOW_HASH : public RecordExt {
+    static int REGISTERED_ID;
+
+    // Value in host byte order
+    uint64_t flow_hash;
+
+    RecordExtFLOW_HASH() : RecordExt(REGISTERED_ID)
+    {
+        flow_hash = 0;
+    }
+
+#ifdef WITH_NEMEA
+    virtual void fill_unirec(ur_template_t *tmplt, void *record)
```

Comment:
```suggestion
   void fill_unirec(ur_template_t *tmplt, void *record) override
```

Reply:
maybe this should be also fixed in the plugin template